docs/eta-pi-p/index.md

```diff
@@ -4,22 +4,17 @@
 
 ### CAS-assisted Amplitude Models building for photoproduction reaction
 
-This document is a follow-up of PWA101 (v1.0) (temporarily see [here](https://compwa--217.org.readthedocs.build/report/999.html)),
-as an introduction to Amplitude Analysis / Partial Wave Analysis (PWA) by demonstrating its application to a specific reaction channel and amplitude model.
+This document is a follow-up of PWA101 (v1.0), see **[TR&#8209;033](https://compwa.github.io/report/033)**.
 
-Now the focus is the use of symbolic expressions (computations), with the use of [ComPWA packages](https://compwa.github.io/), or CAS(Computer Algebra System)-assisted model building in general (the python package `SymPy` in this case) to illustrate the general process in PWA.
-
-This tutorial aims to equip readers with deeper understanding of PWA methodologies and full workflow in the context of hadron physics through a practical, hands-on example with symbolic expressions via [ComPWA](https://compwa.github.io/).
-
-Furthermore,
-address to the issue ✅&nbsp;[ComPWA/gluex-nstar#1](https://github.com/ComPWA/gluex-nstar/issues/1), this document is PWA101(v2.0), which shows PWA methodologies and full workflow in the context of hadron physics with symbolic expressions via [ComPWA](https://compwa.github.io/).
+Now the focus is the use of symbolic expressions (computations), with the use of [ComPWA packages](https://compwa.github.io/), or CAS(Computer Algebra System)-assisted model building in general (the python package SymPy in this case) to illustrate the general process in PWA.
+Address to the issue ✅&nbsp;[ComPWA/gluex-nstar#1](https://github.com/ComPWA/gluex-nstar/issues/1), this document is PWA101(v2.0), which shows PWA methodologies and full workflow in the context of hadron physics with symbolic expressions via `sympy` and [ComPWA](https://compwa.github.io/).
 
 In the following, an attempt to manually formulate the amplitude model via Sympy is shown.
 Later on, attempts of automated amplitude model formualtion are done via e.g. `ampform` and `ampform-dpd`.
 
 ```{toctree}
 reaction-model
-pgamma-state
 manual
```

Comment:
Why did the order change?

Reply:
Because reaction-model is for the flexible model implemented in `sympy`, so this order is more logical. And `ampform` one now goes after `qrules`